modules/@apostrophecms/page/index.js

```diff
@@ -2497,6 +2491,48 @@ database.`);
             await self.apos.attachment.recomputeAllDocReferences();
           }
         });
+      },
+      async inferLastTargetIdAndPosition(doc) {
```

Comment:
This code was borrowed from an old migration that calculated these properties when missing, which was in turn removed because we no longer store these properties.